.github/workflows/manual-testing.yml

```diff
@@ -0,0 +1,148 @@
+    name: Token Create Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: TokenCreateContract
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  TokenQueryContract:
+    name: Token Query Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: TokenQueryContract
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  TokenManagmentContract:
+    name: Token Managment Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: TokenManagmentContract
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  TokenTransferContract:
+    name: Token Transfer Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: TokenTransferContract
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  Proxy:
+    name: Proxy Upgrade Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: Proxy
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  SafeHTS:
+    name: SafeHTS Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: SafeHTS
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  HIP583:
+    name: HIP583 Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: HIP583
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  Multicall:
+    name: Multicall Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: Multicall
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  HRC:
+    name: HRC Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: HRC
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  ShanghaiOpcodes:
+    name: ShanghaiOpcodes Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: ShanghaiOpcodes
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
+
+  PrngSystemContract:
+    name: PrngSystemContract Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: PrngSystemContract
+      networkTag: ${{inputs.networkNodeTag}}
+      mirrorTag: ${{inputs.mirrorNodeTag}}
+      relayTag: ${{inputs.relayTag}}
```

Comment:
Do you think we should make some room for the Solidity, Yul, and OZ contracts as well? https://github.com/hashgraph/hedera-smart-contracts/blob/main/.github/workflows/solidity-tests.yml

Reply:
Might be too much for this case but not a bad idea.
@georgi-l95 have you tried the run out?
How long does everything take already and then when you add the above suggestions?

Reply:
Yep, we can add them, they run in parallel, so it shouldn't run longer than now, which is as long as the slowest test (around 5 minutes)